core/indigo-core/molecule/molecule_allene_stereo.h

```diff
@@ -86,7 +86,7 @@ namespace indigo
 
         bool _isAlleneCenter(BaseMolecule& mol, int idx, _Atom& atom, int* sensible_bonds_out);
 
-        RedBlackMap<int, _Atom> _centers;
+        std::map<int, _Atom> _centers;
```

Comment:
That's what we discussed. We can't use standard containers as members of classes that are used inside Indigo memory-based containers. It leads to memory corruption. That's why the tests failed.